cypress/README.md

```diff
@@ -24,21 +24,26 @@ Use scenario tags:
 3. `@ignore` for temporary ignored scenarios
 4. `@[bug-number]` for example `@FE-1234` - use this tag after `@ignore` to explain why test is ignored.
 5. `@ignore` and `@[bug-number]` should be removed after bug fix
+6. `@build` for pipeline - each build is veirfied by this tests
+7. `@sanity` tests verifies accessibility violations
 
 ## Files structure
 ```
 .
 ├── cypress
 │ ├── fixture
-│ ├── integration
+│ ├── features
+│ │   ├── build
+│ │   ├── sanity
```

Comment:
sanity package is under regression :)

Reply:
I use this order to show status of tests:
build/pipeline - build folder
sanity - Accessibility section - more than build
regression - all other tests